### DISCUSSION TITLE
Simple-very simple- suggestions for general usage

### DISCUSSION THREAD

**Discussion by linfamonte** [xournalpp/xournalpp#5883] [Ideas]
I'm using the app from very little time but I'm finding it very fluid and the only thing that I'm missing right now are 2:

bucket and stroke in separated options, if you want sometimes fill something separately from the drawing, and sometimes to fill the entire page in a more practical way;

COLORS! Why don't we have still the way to make a custom palette of colors? Even Paint allows that. (additional options could be also: create multiple lists of colors and take them as needed- replace permanently the standard color palette, good but generic) and make them fixed if we close the app, so we can just set the panel in a more durable way.

**Comment by rolandlo** [comment]
> I'm using the app from very little time but I'm finding it very fluid and the only thing that I'm missing right now are 2:
> 
> bucket and stroke in separated options, if you want sometimes fill something separately from the drawing, and sometimes to fill the entire page in a more practical way;

With "bucket" you mean a fill tool for regions defined by previously drawn lines? Or just the fill option? I don't quite get it.

> 
> COLORS! Why don't we have still the way to make a custom palette of colors? Even Paint allows that. (additional options could be also: create multiple lists of colors and take them as needed- replace permanently the standard color palette, good but generic) and make them fixed if we close the app, so we can just set the panel in a more durable way.

In the [nightly build](https://github.com/xournalpp/xournalpp/releases/tag/nightly) you can define custom palettes of colors, see [website](https://xournalpp.github.io/guide/config/toolbar-colors/#toolbar-palette-files), and they are accessible in the GUI (Palette tab in menu `Edit > Preferences`)

**Comment by linfamonte** [comment]
Hi, and thank you for the answer. With "bucket" I meant this, the fill option, but as a standalone tool.
